assets/js/membraneWebRTC.ts

```diff
@@ -320,16 +366,20 @@ export class MembraneWebRTC {
           (trans) =>
             (trans.direction = trans.direction == "sendrecv" ? "sendonly" : trans.direction)
         );
-
-      let mediaEvent = generateMediaEvent("renegotiateTracks", {});
-      this.sendMediaEvent(mediaEvent);
     }
+    let mediaEvent = generateMediaEvent("renegotiateTracks", {});
+    this.sendMediaEvent(mediaEvent);
```

Comment:
Why did we move this out of the `if (this.connection)`

Reply:
It handle situation when we join empty room and later want add new track. Because we didn't receive offerData earlier we need to send renegotiateTracks to receive offerData and created connection.

Reply:
Wouldn't be easier to create `peerConnection` in membraneWebRTC constructor?

Reply:
I think it is not good idea because after cleanup it will be impossible to create again peerConnection